static/styles/zulip.css

```diff
@@ -1399,16 +1399,6 @@ td.pointer {
     height: 100%;
 }
 
-.last_message {
-    .unread_marker {
-        bottom: 0;
-    }
-
-    .messagebox-content {
-        padding-bottom: 1px;
-    }
-}
-
```

Comment:
We should do a follow-up commit to remove the `last_message` computation entirely, I think?

Reply:
I'll do this and merge shortly -- started on it and realized there's a weird lightbox code path to clean up here.